exe/pushmi_pullyu

```diff
@@ -2,4 +2,13 @@
 
 require 'pushmi_pullyu'
 
-PushmiPullyu::CLI.new(ARGV).run!
+begin
+  cli = PushmiPullyu::CLI.instance
+  cli.parse
+  cli.run
+rescue => e
+  raise e if $DEBUG
+  STDERR.puts e.message
```

Comment:
Are we just going to log to STDERR, or are we deferring logging to a later ticket?